biome.jsonc

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "https://biomejs.dev/schemas/2.0.6/schema.json",
+  "root": true,
+  "vcs": {
+    "enabled": false,
+    "clientKind": "git",
+    "useIgnoreFile": false,
+  },
+  "files": {
+    "ignoreUnknown": false,
+    "includes": [
+      "**/*.{js,ts}",
+
+      // test result artifacts
+      "!coverage/**",
+
+      // release build artifacts
+      "!dist/**",
+      "!dist-gha/**",
```

Comment:
I thought with `vcs.useIgnoreFile: true` we could get rid of these and have it automatically exclude files in our `.gitignore` file but it didn't work.

Reply:
this feels like a pretty rough oversight and doesn't inspire confidence tbh 😔 but glad the workaround is easy enough at least. mind adding a comment explaining this?